Sources/BraintreeDropIn/Models/BTDropInResult.m

```diff
@@ -11,30 +11,67 @@
 #import <BraintreeCore/BraintreeCore.h>
 #endif
 
+NSString *const BTDropInResultErrorDomain = @"com.braintreepayments.BTDropInResultErrorDomain";
+
 @implementation BTDropInResult
 
 #pragma mark - Prefetch BTDropInResult
 
-+ (void)fetchDropInResultForAuthorization:(NSString *)authorization handler:(BTDropInResultFetchHandler)handler {
-    BTUIKPaymentOptionType lastSelectedPaymentOptionType = [[NSUserDefaults standardUserDefaults] integerForKey:@"BT_dropInLastSelectedPaymentMethodType"];
-    __block BTAPIClient *apiClient = [[BTAPIClient alloc] initWithAuthorization:authorization sendAnalyticsEvent:NO];
+static NSUserDefaults *_userDefaults = nil;
+
++ (NSUserDefaults *)userDefaults {
+    if (_userDefaults == nil) {
+        _userDefaults = [NSUserDefaults standardUserDefaults];
+    }
+    return _userDefaults;
+}
+
++ (void)setUserDefaults:(NSUserDefaults *)userDefaults {
```

Comment:
random: visible for testing?

Reply:
Yep! That's specified for this property in `BTDropInResult_Internal.h`.